src/content/docs/ko/tutorial/1-setup/1.mdx

```diff
@@ -0,0 +1,116 @@
+---
+type: tutorial
+title: 개발 환경 준비
+description: |-
+  튜토리얼: 첫 번째 Astro 블로그 구축 - 튜토리얼을 완료하는 데 필요한 로컬 도구 설치
+i18nReady: true
+---
+import Badge from '~/components/Badge.astro';
+import Checklist from '~/components/Checklist.astro';
+import Box from '~/components/tutorial/Box.astro';
+import MultipleChoice from '~/components/tutorial/MultipleChoice.astro';
+import Option from '~/components/tutorial/Option.astro';
+import PreCheck from '~/components/tutorial/PreCheck.astro';
+
+<PreCheck>
+  - Astro 웹 사이트를 구축하는 데 사용할 도구를 설치하십시오.
+</PreCheck>
+
+## 필요한 개발 도구를 확보하세요
+
+### 터미널
+
+**명령줄 (터미널)**을 사용하여 Astro 프로젝트를 만들고 주요 명령을 실행하여 사이트를 빌드, 개발, 테스트하게 됩니다.
```

Comment:
Seems like the bold syntax failed here, could you give a look? I guess a space should do the trick.

![image](https://github.com/withastro/docs/assets/61414485/026580ff-8ce2-478b-a704-7411129235ef)

Reply:
Fixed (I think). Good enough  :smile:

Reply:
There was a mistake. Thanks for fixing it!